athena-gcs/src/main/java/com/amazonaws/athena/connectors/gcs/GcsExceptionFilter.java

```diff
@@ -0,0 +1,40 @@
+/*-
+ * #%L
+ * Amazon Athena GCS Connector
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.gcs;
+
+import com.amazonaws.athena.connector.lambda.ThrottlingInvoker;
+import com.amazonaws.services.athena.model.AmazonAthenaException;
+
+public class GcsExceptionFilter implements ThrottlingInvoker.ExceptionFilter
```

Comment:
This Exception filter is explicitly for throttling. Can we rename it to something meaningful? Like GcsThrottlingExceptionFilter

Reply:
taken care. changes pushed.